### PR TITLE
Fix documentation of GA is in APIService's PriorityVersion and CRD's Versions

### PR DESCRIPTION
**Pull request by mbohlool**
Explain what GA is in APIService's PriorityVersion and CRD's Versions.

ref #64136 

```release-note
NONE
```

### CODE REVIEW

**Comment by sttts** [comment]
Sgtm, but leaving the final lgtm to a native speaker (@deads2k @liggitt ?)

**Comment by liggitt** [comment]
Works for me

File: staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go [hunk 1]
```diff
@@ -45,8 +45,9 @@ type CustomResourceDefinitionSpec struct {
 	// If the version string is "kube-like", it will sort above non "kube-like" version strings, which are ordered
 	// lexicographically. "Kube-like" versions start with a "v", then are followed by a number (the major version),
 	// then optionally the string "alpha" or "beta" and another number (the minor version). These are sorted first
-	// by GA > beta > alpha, and then by comparing major version, then minor version. An example sorted list of
-	// versions: v10, v2, v1, v11beta2, v10beta3, v3beta1, v12alpha1, v11alpha2, foo1, foo10.
+	// by GA > beta > alpha (where GA is a version with no suffix such as beta or alpha), and then by comparing
```
[the remark this answers is not in the document]
Comment (mbohlool) [reply staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go]: I don't have strong feeling here too. If you are OK, I will merge this one for now as Jordan is OK with it too.

**Comment by mbohlool** [comment]
@liggitt I think I will go with the existing one for now. I don't have a strong feeling about it, however.

Can you lgtm approve this?

**Pull request merged**